data/json/itemgroups/Weapons_Mods_Ammo/ammo.json

```diff
@@ -766,6 +766,13 @@
       { "item": "shot_hull", "prob": 50, "count": [ 1, 3 ] }
     ]
   },
+  {
+    "id": "ammo_casings_gunsmith",
+    "//": "Casings found in prepper/gunsmith locations, including primed casings",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [ { "group": "ammo_casings", "prob": 98 }, { "item": "22_casing_new", "prob": 2 } ]
```

Comment:
What's the point to make a new group to add 22 instead of adding it directly to ammo_casings?

Reply:
It being in `ammo_casings` causes the bug.  The `field` and `road` item groups use the `ammo_casings` group to portray where a human was killed by gunfire during the events of the cataclysm, an unused casing shouldn't appear in that situation.